src/gyp.py

```diff
@@ -0,0 +1,8 @@
+# Dummy gyp.py because chromium/src/vs_toolchain.py depends on gyp, but it's in
```

Comment:
The ugliest bit is that the vs_toolchain.py script imports from some gyp-related code that lives in a different repo. We don't actually use that, so we stub it out. Another option is to use the version of that script in v8 (`v8/gypfiles/vs_toolchain.py`) which duplicates that functionality but doesn't depend on gyp. In which case we wouldn't need the fake gyp, but all the CMake builds would depend on the v8 build.